### PR TITLE
Send slow command to statsd

### PR DESCRIPTION
**Pull request by doyoubi**

### CODE REVIEW

File: src/connection.c [hunk 1]
```diff
@@ -75,6 +75,8 @@ static struct connection *conn_create_server(struct context *ctx,
     struct connection *server = server_create(ctx, fd);
     struct conn_info *info = server->info;
     memcpy(&info->addr, addr, sizeof(info->addr));
+    extern const size_t CMD_NUM;
+    info->slow_cmd_counts = cv_calloc(CMD_NUM, sizeof(uint32_t));
```
Comment (wooparadog) [inline src/connection.c]: If only server connection should have this field, maybe we should get it out of `struct connection`?

File: src/slowlog.c [hunk 2]
```diff
@@ -9,6 +9,15 @@
 static uint32_t slowlog_id = 0;
 static inline size_t min(size_t a, size_t b) { return a < b ? a : b; }
 
+extern const size_t CMD_NUM;
+
+struct dict slow_counts;  // node dsn => slow cmd counts
```
Comment (wooparadog) [inline src/slowlog.c]: Better naming?

File: src/connection.c (hunk 1 above)
Comment (doyoubi) [inline src/connection.c]: Actually `struct connection` is used to represent three kinds of objects, client connection, server connection, and event handler. But they do share some fields so I think putting them together is ok.

File: src/slowlog.c (hunk 2 above)
Comment (doyoubi) [inline src/slowlog.c]: What about `slow_cmd_table`?

**Comment by tdsparrow** [comment]
Looks good.

**Pull request merged**